src/screens/Dashboard/Dashboard.js

```diff
@@ -286,31 +313,31 @@ export default function HrDashboard(props) {
   };
   const [approveRequestModal, setApproveRequestModal] = useState({
     show: false,
-    data: null,
+    data: null
   });
 
   const [historyModal, setHistoryModal] = useState({
     show: false,
-    data: null,
+    data: null
   });
 
   const [regularizationRequest, setRegularizationRequest] = useState([]);
   const [ticketID, setTicketID] = useState();
   const [isLoading, setIsLoading] = useState(false);
   const [notificationId, setNotificationId] = useState();
 
-  const handleRegularizationRequest = (cuurentData) => {
-    
-    
+  const handleRegularizationRequest = async (currentData) => {
+    const id =
+      currentData?.ticketID === undefined ? currentData : currentData.ticketID;
     setIsLoading(null);
     setIsLoading(true);
-    setTicketID(cuurentData.ticketID ? cuurentData.ticketID : cuurentData);
-    setNotificationId(cuurentData.notificationid);
-    new getRegularizationTime(ticketID).then((res) => {
+    setTicketID(id);
+    setNotificationId(currentData.notificationid);
+    await new getRegularizationTime(id).then((res) => {
```

Comment:
Currently , for eg
id is goin as "1652" in payload ,
But in case currentData doesnt have ticketID than payload will go as {ticketId : "" , NotificationId "xxxxx"};
is it handled by Backend team?
If not than it will crash

Reply:
informed to yogita